.github/workflows/ci.yml

```diff
@@ -119,6 +119,9 @@ jobs:
           bash -x start_ocean.sh --with-provider2 --no-dashboard --with-c2d 2>&1 > start_ocean.log &
         env:
           CONTRACTS_VERSION: v1.0.0-alpha.28
+          PROVIDER_VERSION: refactor_signatures
+          AQUARIUS_VERSION: refactor_signatures
+          OPERATOR_SERVICE_VERSION: refactor_signatures
```

Comment:
we shouldn't have this branches here.

Reply:
Will change that as soon as we merge them and release new versions.
If I will change it now, all tests are going to fail.
We need to merge all PRs for all components in the same time